src/azure-cli/azure/cli/command_modules/dms/custom.py

```diff
@@ -42,14 +43,15 @@ def check_service_name_availability(client, service_name, location):
                                           parameters=parameters)
 
 
-def create_service(client,
+def create_service(cmd,
+                   client,
                    service_name,
                    resource_group_name,
-                   location,
                    subnet,
                    sku_name,
                    tags=None,
                    no_wait=False):
+    location = get_rg_location(cmd.cli_ctx, resource_group_name)
```

Comment:
A resource group can include resources from other locations. For example, I have multiple DMS services in Central US in the resource group that was initially created in West US 2. While this change may simplify the command-line parameters, the users will not be able create a service which is not in the resource group's home location. This is a limiting change. Do we want that?